ee/codegen/src/__test__/nodes/__snapshots__/conditional-node.test.ts.snap

```diff
@@ -39,6 +51,7 @@ class ConditionalNodeDisplay(BaseConditionalNodeDisplay[ConditionalNode]):
             "cf63d0ad-5e52-4031-a29f-922e7004cdd8"
         ),
     }
+    port_displays = {}
```

Comment:
[q] Do we expect this to be empty?

Reply:
Yes because I didn't pass in any edges for the workflow data factory